sources/packages/forms/src/form-definitions/institutionuserprofile.json

```diff
@@ -114,18 +134,15 @@
       "defaultValue": null,
       "protected": false,
       "unique": false,
-      "persistent": true,
-      "clearOnHide": true,
+      "persistent": false,
```

Comment:
What change has made these properties false? asking for my understanding.

Reply:
I don't think the changed line showing here belongs to this component. this new panel was added by me.
old form 
![image](https://user-images.githubusercontent.com/77353155/190498553-ad8aa762-8904-4d6d-b4bf-65474f6a4c0b.png)
new form 
![image](https://user-images.githubusercontent.com/77353155/190498628-4a31616e-2861-4801-98fa-aa349586f653.png)